package.json

```diff
@@ -37,14 +37,14 @@
     "jest": "^26.1.0",
     "opener": "^1.5.2",
     "prettier": "^2.0.5",
-    "shx": "^0.3.3",
     "ts-jest": "^26.1.3",
     "ts-node": "^9.1.1",
     "typescript": "^3.9.7"
   },
   "scripts": {
-    "build": "shx rm -rf ./build && tsc",
-    "build:watch": "shx rm -rf ./build && tsc -w",
+    "build": "tsc",
+    "build:watch": "tsc -w",
+    "cli": "ts-node --files src/cli.ts",
```

Comment:
We should probably add this to the README so that first-time users are aware that they can run `npm run cli`